the nvidia drivers are trying to test for version 2.6.6, but fail to take into
account the first digit.  so 3.7.0+ ends up failing the test.

https://bugs.gentoo.org/447566

--- a/NVIDIA-Linux-x86_64-313.18-no-compat32/kernel/conftest.sh	2013-02-04 15:10:20.669633666 -0500
+++ b/NVIDIA-Linux-x86_64-313.18-no-compat32/kernel/conftest.sh	2013-02-04 15:10:34.283334673 -0500
@@ -1697,13 +1697,15 @@
                 # kernel older than 2.6.6, that's all we require to
                 # build the module.
                 #
+                VERSION=$(grep "^VERSION =" $MAKEFILE | cut -d " " -f 3)
                 PATCHLEVEL=$(grep "^PATCHLEVEL =" $MAKEFILE | cut -d " " -f 3)
                 SUBLEVEL=$(grep "^SUBLEVEL =" $MAKEFILE | cut -d " " -f 3)
 
-                if [ -n "$PATCHLEVEL" -a $PATCHLEVEL -ge 6 \
+                if [ -n "$VERSION" -a $VERSION -ge 3 ] || \
+                   [ -n "$PATCHLEVEL" -a $PATCHLEVEL -ge 6 \
                         -a -n "$SUBLEVEL" -a $SUBLEVEL -le 5 ]; then
                     SELECTED_MAKEFILE=Makefile.kbuild
                     RET=0
                 fi
             fi
         fi
